concepticondata/conceptlists/Nagano-2013-1256.tsv

```diff
@@ -0,0 +1,1257 @@
+Nagano-2013-1256-836	836	1485	rabbit	1136	RABBIT	
+Nagano-2013-1256-837	837	1487	mouse	1491	MOUSE	
+Nagano-2013-1256-838	838	1489	elephant	1290	ELEPHANT	
+Nagano-2013-1256-839	839	1491	camel	1871	CAMEL	
+Nagano-2013-1256-840	840	1493	bat	1793	BAT	
+Nagano-2013-1256-841	841	1495	eagle	1905	EAGLE	
+Nagano-2013-1256-842	842	1497	crane	1830	CRANE	
+Nagano-2013-1256-843	843	1499	swallow	3077	SWALLOW (BIRD)	
+Nagano-2013-1256-844	844	1500	little cuckoo	2934	CUCKOO	
+Nagano-2013-1256-845	845	1502	pigeon	1853	DOVE	
+Nagano-2013-1256-846	846	1503	pigeon	1853	DOVE	
+Nagano-2013-1256-847	847	1505	crow	913	CROW	
+Nagano-2013-1256-848	848	1506	crow	913	CROW	
+Nagano-2013-1256-849	849	1507	owl	735	OWL	
+Nagano-2013-1256-850	850	1509	sparrow	1854	SPARROW	
+Nagano-2013-1256-851	851	1511	butterfly	1791	BUTTERFLY	
+Nagano-2013-1256-852	852	1513	ant	587	ANT	
+Nagano-2013-1256-853	853	1515	spider	843	SPIDER	
+Nagano-2013-1256-854	854	1517	honey-bee	665	BEE	
+Nagano-2013-1256-855	855	1519	mosquito	1509	MOSQUITO	
+Nagano-2013-1256-856	856	1521	fly			
+Nagano-2013-1256-857	857	1523	flea	232	FLEA	
+Nagano-2013-1256-858	858	1525	louse	1392	LOUSE	
+Nagano-2013-1256-859	859	1527	snake	730	SNAKE	
+Nagano-2013-1256-860	860	1529	lizard	632	LIZARD	
+Nagano-2013-1256-861	861	1530	frog	503	FROG	
+Nagano-2013-1256-862	862	1532	horn	1393	HORN (ANATOMY)	
+Nagano-2013-1256-863	863	1534	hair			
+Nagano-2013-1256-864	864	1535	mane	1918	MANE	
+Nagano-2013-1256-865	865	1537	skin, fur	580	FUR	
+Nagano-2013-1256-866	866	1539	claw	72	CLAW	
+Nagano-2013-1256-867	867	1540	hoof	152	HOOF	
+Nagano-2013-1256-868	868	1542	tail	1220	TAIL	
+Nagano-2013-1256-869	869	1543	tail (long)			
+Nagano-2013-1256-870	870	1545	beak	73	BEAK	
+Nagano-2013-1256-871	871	1547	wing	1257	WING	
+Nagano-2013-1256-872	872	1549	feather	1201	FEATHER	
+Nagano-2013-1256-873	873	1551	scales of fish	266	SCALE	
+Nagano-2013-1256-874	874	1552	scales of snake			
+Nagano-2013-1256-875	875	1554	nest	539	NEST	
+Nagano-2013-1256-876	876	1556	fly (v)	1441	FLY (MOVE THROUGH AIR)	
+Nagano-2013-1256-877	877	1558	swim (v)	1439	SWIM	
+Nagano-2013-1256-878	878	1560	cry (v)			
+Nagano-2013-1256-879	879	1562	lay an egg (v)	2464	LAY EGGS	
+Nagano-2013-1256-880	880	1564	raise (v)			
+Nagano-2013-1256-881	881	1567	shape	2134	RAISE (A CHILD)	
+Nagano-2013-1256-882	882	1569	round	1395	ROUND	
+Nagano-2013-1256-883	883	1570	round	1395	ROUND	
+Nagano-2013-1256-884	884	1572	sharp	1396	SHARP	
+Nagano-2013-1256-885	885	1574	blunt	379	BLUNT	
+Nagano-2013-1256-886	886	1576	level and smooth	1234	SMOOTH	
+Nagano-2013-1256-887	887	1577	flat	1633	FLAT	
+Nagano-2013-1256-888	888	1578	flat	1633	FLAT	
+Nagano-2013-1256-889	889	1580	hole	1667	HOLE	
+Nagano-2013-1256-890	890	1582	hollow			
+Nagano-2013-1256-891	891	1584	straight	1404	STRAIGHT	
+Nagano-2013-1256-892	892	1585	straight (adv)	1404	STRAIGHT	
+Nagano-2013-1256-893	893	1587	line	1048	LINE	
+Nagano-2013-1256-894	894	1589	marks			
+Nagano-2013-1256-895	895	1590	marks			
```

Comment:
Doubled again. Just reporting in case this is a problem, please ignore if it isn't. :)